src/games/strategy/net/MacFinder.java

```diff
@@ -36,41 +41,26 @@ private static String GetMacAddress() {
       final InetAddress address = InetAddress.getLocalHost();
       final NetworkInterface localHostNI = NetworkInterface.getByInetAddress(address);
       if (localHostNI != null) {
-        // final byte[] rawMac = localHostNI.getHardwareAddress();
-        final Method m = NetworkInterface.class.getMethod("getHardwareAddress");
```

Comment:
This is more than a refactor here. How do you know this is safe, and is not used for Java RPC? Could have implications for network games, but perhaps would work fine for single player